src/components/NcAppNavigationCaption/NcAppNavigationCaption.vue

```diff
@@ -88,7 +88,7 @@ export default {
 .app-navigation-caption {
 	display: flex;
 	justify-content: space-between;
-	padding: 0 8px 0 math.div($clickable-area, 2);
+	padding: 0 8px 0 12px;
```

Comment:
Would be better to use the variable --default-grid-baseline here, in these cases *2 and *3.

Reply:
Would be nice to have it dependent on a CSS variable. However, `--default-grid-baseline` is only `4px` and the caption would be a bit too far left.

IMO, it should be `(--default-clickable-area - 20px) / 2 = 12px`. Our default icon width/size in the sidebar is `20px` hence the subtraction.

Reply:
As said, "in these cases *2 and *3". :) Everything should be multiples of --default-grid-baseline, so here:
```suggestion
	padding: 0 calc(var(--default-grid-baseline, 4px) * 2) 0 calc(var(--default-grid-baseline, 4px) * 3);
```

Reply:
@juliushaertl @skjnldsv is that documented anywhere in the Vue styleguide btw? Might be good to have that :)

Reply:
Ahhh, thanks Jan. Learnt something new today :)

Reply:
@st3iny thanks go to @juliushaertl, he added the variable and I also learned then that variables can have backup values. :D